spec/dummy/config/locales/cy.yml

```diff
@@ -0,0 +1,2 @@
+cy:
```

Comment:
Yeah, otherwise when specs run it complains about there being no "cy" locale. Any ideas around that?